crates/provider/src/native.rs

```diff
@@ -61,7 +54,7 @@ impl<T: FileSystem + Send + Sync> NativeProvider<T> {
         tmp_dir: impl Into<String>,
         filesystem: T,
     ) -> Self {
-        let e = std::io::Error::new(std::io::ErrorKind::BrokenPipe, "error");
+        let _e = std::io::Error::new(std::io::ErrorKind::BrokenPipe, "error");
```

Comment:
```suggestion
```
don't used, could be something that was a test or something?

Reply:
Could be. Since it was your wip I just added the `_` so that I wont remove code that you may need